api/metrics-api-jaxrs/pom.xml

```diff
@@ -199,6 +199,30 @@
         <!-- Document generation from the Swagger annotations on the REST-API. -->
         <plugins>
           <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>unpack</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>unpack</goal>
+                </goals>
+                <configuration>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>org.hawkular</groupId>
+                      <artifactId>hawkular-build-tools</artifactId>
+                      <version>${version.org.hawkular.hawkular-build-tools}</version>
```

Comment:
Ha, right, it's not a dependency, it's an artifactItem (so version can't be
inherited from dependency management)

2016-04-26 12:12 GMT+02:00 Pavol Loffay <notifications@github.com>:

> In api/metrics-api-jaxrs/pom.xml
> <https://github.com/hawkular/hawkular-metrics/pull/486#discussion_r61061256>
> :
>
> > @@ -199,6 +199,30 @@
> >          <!-- Document generation from the Swagger annotations on the REST-API. -->
> >          <plugins>
> >            <plugin>
> > +            <groupId>org.apache.maven.plugins</groupId>
> > +            <artifactId>maven-dependency-plugin</artifactId>
> > +            <executions>
> > +              <execution>
> > +                <id>unpack</id>
> > +                <phase>process-resources</phase>
> > +                <goals>
> > +                  <goal>unpack</goal>
> > +                </goals>
> > +                <configuration>
> > +                  <artifactItems>
> > +                    <artifactItem>
> > +                      <groupId>org.hawkular</groupId>
> > +                      <artifactId>hawkular-build-tools</artifactId>
> > +                      <version>${version.org.hawkular.hawkular-build-tools}</version>
>
> ${version.org.hawkular.hawkular-build-tools} is defined in hawkular-parent
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hawkular/hawkular-metrics/pull/486/files/e9cfda67b03482154df2f0834e0cff6c17ec3d17#r61061256>
>



-- 
Thomas Segismont
JBoss ON Engineering Team